public/data_load/kibana_cloned_code/courier.ts

```diff
@@ -186,21 +197,21 @@ export const handleCourierRequest = async ({
 
   if (shouldCalculateNewTabify) {
     (searchSource as any).lastTabifyHash = tabifyCacheHash;
-    (searchSource as any).tabifiedResponse = search.tabifyAggResponse(
-      aggs,
+    (searchSource as any).tabifiedResponse = tabifyAggResponse(aggs,
       (searchSource as any).finalResponse,
       tabifyParams
     );
   }
 
-  inspectorAdapters.data.setTabularLoader(
+  // DataAdapter is not present in Kibana 7.12. Is this necessary?
+/*   inspectorAdapters.data.setTabularLoader(
     () =>
       buildTabularInspectorData((searchSource as any).tabifiedResponse, {
         queryFilter: filterManager,
         deserializeFieldFormat: getFieldFormats().deserialize,
       }),
     { returnsFormattedValues: true }
-  );
+  ); */
```

Comment:
After numerous functional tests, it works great, especially "Inspect" feature.
So you can definitively remove this commented block.

Reply:
I have removed the whole block as requested